CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Added a new validation (`PA137`) to **validate**, ensuring the absence of non-ig
 * Updated the **format** command to use the content graph instead of the id_set file.
 * Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
 * Removed a redundant debug log on the `tools.get_file` function.
+* Added the *auto-replace-uuids* flag to the **download** command. Use it to avoid UUID replacements when downloading using download command.
```

Comment:
```suggestion
* Added the *auto-replace-uuids* flag to the **download** command. set this flag to False to avoid UUID replacements when downloading using download command.
```